core/controllers/library.py

```diff
@@ -117,18 +117,22 @@ def get(self):
                 'activity_summary_dicts': recently_published_summary_dicts,
                 'categories': [],
                 'header': feconf.LIBRARY_CATEGORY_RECENTLY_PUBLISHED,
+                'show_view_all_button': True,
```

Comment:
I think a better name for this is something like "is_expandable" or "has_full_results_page", so that it can be referenced like "group.is_expandable" or "group.has_full_results_page", which makes sense conceptually (in that you would expect a boolean result). Starting with a verb like "show_" suggests that this is a method.